background/imports/util.js

```diff
@@ -1,5 +1,12 @@
+// REMINDER: update similar code at /webpages/settings/index.js
```

Comment:
Is there a reason for the duplicate code?

Reply:
Probably not, but I'm not going to fix it yet. I'm just adding a comment for now.